habitat/datasets/rearrange/sim_utilities.py

```diff
@@ -0,0 +1,838 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os.path as osp
+from typing import Any, Dict, List, Optional, Union
+
+import magnum as mn
+import numpy as np
+
+import habitat_sim
+
+
+class DebugVisualizer:
```

Comment:
This looks like cool functionality that could be useful outside of just rearrange dataset generation. How about make this class its own file that doesn't live under `habitat/datasets`?

And how about class documentation?

Reply:
Yeah, I meant to flesh this out a bit more. Not sure if it shouldn't live in the habitat-sim with some of the other utilities.

Reply:
Opted to move this into `habitat/sims/habitat_simulator` to group it with other functionality that leverages and requires a habitat-sim installation.